src/test/java/org/example/SearchField.java

```diff
@@ -1,5 +1,81 @@
 package org.example;
 
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import java.util.List;
+
+
 public class SearchField {
```

Comment:
Best practice is to add the Tests suffix to a class that contains @Tests -> SearchFieldTests